apps/dashboard/app/views/batch_connect/sessions/index.html.erb

```diff
@@ -36,7 +36,7 @@ locals: {
     </div>
     <div class="col-md-9">
       <div id="batch_connect_sessions">
-        <%= render(partial: 'panels', locals: { use_template: false }) %>
+        <%= render(partial: 'panels') %>
```

Comment:
Not really a part of this js refactor, just happened to notice that this is unused.